src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs

```diff
@@ -325,7 +326,15 @@ public IEnumerable<OneWayCompatibilityMappingEntry> CompatibilityMappings
                             new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.Windows, FrameworkConstants.MaxVersion)),
                             new FrameworkRange(
                                 new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.WinRT, FrameworkConstants.EmptyVersion),
-                                new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.WinRT, new Version(4, 5, 0, 0))))
+                                new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.WinRT, new Version(4, 5, 0, 0)))),
+
+                        // NetCoreApp projects support NetStandard
+                        new OneWayCompatibilityMappingEntry(new FrameworkRange(
+                            new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, new Version(1, 0, 0, 0)),
+                            new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, FrameworkConstants.MaxVersion)),
+                            new FrameworkRange(
+                                FrameworkConstants.CommonFrameworks.NetStandard10,
```

Comment:
This is saying netcoreapp 1.0 and greater is supporting up netstandard 1.5 (that is, netstandard 1.0, 1.1, 1.2, 1.3, 1.4, and 1.5 in practice).

If, in the futurue, we want to add netcoreapp 1.1 supporting netstandard up to 1.6, we will add an additional mapping, similar to this one.

Does that make sense?